src/ui/image/Image.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useRef } from "react"
+
+import { Loader, Button } from "ui"
+
+import csx from "./Image.scss"
+
+namespace Image {
+    export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+        className?: string;
+        lowQuality: string;
+        fallback?: string;
+    }
+}
+
+const Image = ({ lowQuality, fallback, src, className, width, height, ...props }: Image.Props) => {
+    const [url, setUrl] = useState(lowQuality || src)
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        if (url === lowQuality) loadImage();
+    }, [])
+
+    const loadImage = (): void => {
```

Comment:
What is this fn doing? Does this improve image loading performance?

Reply:
loadImage function eliminates the effect of a lack of image when loading an image with large dimensions. Fallback is by default, the expected image is loaded in the background, then replaced. Otherwise, we wouldn't see any image when loading a large image. This is one of the strategies I have found.